### ISSUE TITLE
[Risk Framework]: Optimism Risk Page Missing Others Group

### ISSUE THREAD

**Issue by rareweasel** [yearn/ydaemon#282]
In the risk framework [page](https://yearn.watch/risk) (Optimism network), there is no an "Others" group.

<img width="1005" alt="image" src="https://github.com/yearn/ydaemon/assets/89795397/8571e1ea-5e68-4c58-8875-09158c77e7ae">

The "Others" group should have all the strategies that are not include in the rest of the groups (check Ethereum network).

For example, The Sync Yearn Farm [website](https://sync.yearn.farm/) is displaying +15 vaults with strategies without risk scores. All of them should be in the Others group.

Basically, all the networks should follow that logic. So, we can easily check when we are missing a group in the risk framework.

**Comment by 0xBasically** [comment]
this is an error all risk groups are live, closing
